tools/nuget/generate_nuspec_for_native_nuget.py

```diff
@@ -105,8 +105,10 @@ def generate_file_list_for_ep(nuget_artifacts_dir, ep, files_list, include_pdbs,
                 if child_file.suffix in [".aar"]:
                     files_list.append('<file src="' + str(child_file) + '" target="runtimes/android/native"/>')
 
-        if child.name == "onnxruntime-ios-xcframework":
-            files_list.append('<file src="' + str(child) + "\\**" '" target="runtimes/ios/native"/>')  # noqa: ISC001
+        if child.name == "onnxruntime-ios":
```

Comment:
where does this "onnxruntime-ios" directory name come from?

Reply:
there's a loop to trim the trailing '-xxx' off each directory name at the end of the extract script. that in general handles version numbers for most of the packages. existing code was trimming onnxruntime-android-aar to onnxruntime-android, so it's somewhat consistent with that to also let it trim onnxruntime-ios-xcframework to onnxruntime-ios. definitely cryptic though

there's is too much implicit stuff between the extraction script logic and the required structure here. would probably be simpler if the extract was done in python here.

Reply:
ah ok, thanks for the explanation.